hydroshare/urls.py

```diff
@@ -1,200 +1,22 @@
-    # "/.html" - so for this case, the template "pages/index.html"
-    # should be used if you want to customize the homepage's template.
-
-    url("^$", "mezzanine.pages.views.page", {"slug": "/"}, name="home"),
-
-    # HOMEPAGE FOR A BLOG-ONLY SITE
-    # -----------------------------
-    # This pattern points the homepage to the blog post listing page,
-    # and is useful for sites that are primarily blogs. If you use this
-    # pattern, you'll also need to set BLOG_SLUG = "" in your
-    # ``settings.py`` module, and delete the blog page object from the
-    # page tree in the admin if it was installed.
-
-    # url("^$", "mezzanine.blog.views.blog_post_list", name="home"),
-
-    # Override Mezzanine URLs here, before the Mezzanine URL include
-    ("^accounts/signup/", "theme.views.signup"),
-
-    # MEZZANINE'S URLS
-    # ----------------
-    # ADD YOUR OWN URLPATTERNS *ABOVE* THE LINE BELOW.
-    # ``mezzanine.urls`` INCLUDES A *CATCH ALL* PATTERN
-    # FOR PAGES, SO URLPATTERNS ADDED BELOW ``mezzanine.urls``
-    # WILL NEVER BE MATCHED!
-
-    # If you'd like more granular control over the patterns in
-    # ``mezzanine.urls``, go right ahead and take the parts you want
-    # from it, and use them directly below instead of using
-    # ``mezzanine.urls``.
-
-    # MOUNTING MEZZANINE UNDER A PREFIX
-    # ---------------------------------
-    # You can also mount all of Mezzanine's urlpatterns under a
-    # URL prefix if desired. When doing this, you need to define the
-    # ``SITE_PREFIX`` setting, which will contain the prefix. Eg:
-    # SITE_PREFIX = "my/site/prefix"
-    # For convenience, and to avoid repeating the prefix, use the
-    # commented out pattern below (commenting out the one above of course)
-    # which will make use of the ``SITE_PREFIX`` setting. Make sure to
-    # add the import ``from django.conf import settings`` to the top
-    # of this file as well.
-    # Note that for any of the various homepage patterns above, you'll
-    # need to use the ``SITE_PREFIX`` setting as well.
-
-    # ("^%s/" % settings.SITE_PREFIX, include("mezzanine.urls"))
-
-)
-
-hydroshare_urls = urlpatterns
 urlpatterns = [
-    url(r'^hydroshare/', include(hydroshare_urls)),
-    url(r'^hydroshare/', include("mezzanine.urls")),
+    url("^mmh-admin/", include(admin.site.urls)),
+    url(r'^accounts/login/$', auth_views.login, name='login'),
     url(r'', include('myhpom.urls', namespace='myhpom')),
 ]
 
-# Adds ``STATIC_URL`` to the context of error pages, so that error
-# pages can use JS, CSS and images.
-handler404 = "mezzanine.core.views.page_not_found"
-handler500 = "mezzanine.core.views.server_error"
+# HMMM....? Shouldn't these be served by nginx for debug False?
+if settings.DEBUG is False:   # if DEBUG is True it will be served automatically
+    urlpatterns += [
+        url(r'^static/(?P<path>.*)$', 'django.views.static.serve', {'document_root': settings.STATIC_ROOT}),
+]
```

Comment:
I would think so...and it looks like it does:

nginx/config-files/hydroshare-nginx.conf#17

Reply:
(there are two nginx configuration templates - one for SSL one w/o...but they both appear to serve /static)

Reply:
OK...so I guess this must be for the local dev case of having DEBUG=False which I now recall was the default until I did something to turn it on. I guess we could leave it (I realize it looks like I added it, really I just moved and properly indented it) and clarify the commenting...